src/test/java/uk/gov/crowncommercial/dsd/api/auth/config/JacksonConfig.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.crowncommercial.dsd.api.auth.config;
+
+import org.springframework.boot.autoconfigure.jackson.Jackson2ObjectMapperBuilderCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Jackson configuration
+ */
+@Configuration
+@Slf4j
+public class JacksonConfig {
```

Comment:
I assume these changes were meant to update the version in `src/main/java` rather than duplicate it..?

Reply:
🤦 how did I miss that! Blind as a bat - muchas gracias!

Reply:
Should be resolved now 👍